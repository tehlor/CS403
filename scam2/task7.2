(println (transpose '((0 0 1 8 9) (4 1 3 5 1) (7 5 1 4 5) (4 0 1 3 8))))
