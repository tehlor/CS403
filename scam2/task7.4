(println (transpose '((9 2 1 5) (8 5 8 5) (3 6 2 4) (3 5 3 3) (3 5 9 9))))
