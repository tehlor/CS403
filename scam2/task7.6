(println (transpose '((8 4 5) (3 7 1) (0 7 5))))
