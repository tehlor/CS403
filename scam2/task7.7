(println (matrix-*-matrix '((5 2 2) (9 2 6) (9 8 6)) '((1 3 1) (6 5 8) (1 1 7) (0 0 8) (5 9 6))))
