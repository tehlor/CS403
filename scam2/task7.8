(println (transpose '((7 6 4) (0 6 2) (2 4 7) (1 8 5))))
