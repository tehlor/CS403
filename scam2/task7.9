(println (matrix-*-matrix '((9 4 6 1 6) (7 6 5 4 7) (8 9 3 6 0)) '((9 8 6) (1 5 2) (6 0 8) (2 9 6))))
