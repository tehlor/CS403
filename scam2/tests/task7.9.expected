((185 138 112 77 110) (60 52 37 33 41) (118 96 60 54 36) (129 116 75 74 75))
